tools/build_pak_files.py

```diff
@@ -0,0 +1,129 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import argparse
+import glob
+import os
+import posixpath
+import spear
+import subprocess
+import sys
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--contents_dir", required=True)
+    parser.add_argument("--unreal_binaries_dir", required=True)
+    parser.add_argument("--platforms", nargs="*", required=True)
+    parser.add_argument("--output_dir", required=True)
+    parser.add_argument("--unreal_project_dir")
```

Comment:
It's fine with me if we hard-code `unreal_project_dir` to `SpearSim`, similar to how we do it in `build_executable.py`. I don't think it would ever happen that we'd want to build everything in `args.contents_dir` for every project in `unreal_projects`.

Reply:
I agree, updated code to handle only `SpearSim`.